src/Storefront/Resources/views/storefront/layout/footer/footer.html.twig

```diff
@@ -80,26 +81,35 @@
                                         {% endif %}
 
                                         {% block layout_footer_navigation_information_icons %}
-                                            <div class="footer-column-toggle">
-                                                <span class="footer-plus-icon">
-                                                    {% sw_icon 'plus' style {
-                                                        ariaLabel: 'general.expand'|trans
-                                                    } %}
-                                                </span>
-                                                <span class="footer-minus-icon">
-                                                    {% sw_icon 'minus' style {
-                                                        ariaLabel: 'general.collapse'|trans
-                                                    } %}
-                                                </span>
-                                            </div>
+                                            {% if root.children|length > 0 %}
+                                                <button class="footer-column-toggle btn btn-link btn-link-inline js-collapse-footer-column-trigger"
+                                                        data-bs-target="#collapseFooterTitle{{ loop.index }}"
+                                                        aria-controls="collapseFooter{{ loop.index }}"
+                                                        aria-labelledby="collapseFooterTitle{{ loop.index }}"
+                                                        aria-expanded="true">
+
+                                                    <span class="footer-plus-icon">
```

Comment:
Do we really need this span? `sw_icon` should already create a wrapping span around the SVG with class `icon-plus` around the icon.

Reply:
Yes, I don't know why it was there before. 🤷‍♂️ Will remove it. 👍